### PR TITLE
DevWs for eZEX containing containing ZigBee Switch  and 1 more

### PR DESCRIPTION
**Pull request by tklee2020**

### CODE REVIEW

File: devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy [hunk 1]
```diff
@@ -37,6 +37,8 @@ metadata {
 		
 		// EZEX
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0006", outClusters: "0006, 000A, 0019", model: "E220-KR1N0Z0-HA", deviceJoinName: "eZEX Switch" //EZEX Switch
+        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0006", outClusters: "0006, 000A, 0019", model: "E220-KR1N0Z1-HA", deviceJoinName: "eZEX Switch" //EZEX Switch
+        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0006", outClusters: "0006, 000A, 0019", model: "E220-KR1N0Z2-HA", deviceJoinName: "eZEX Switch" //EZEX Switch
```
Comment (KKlimczukS) [inline devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy]: Please fix the indentations (change 4 spaces to 1 tab)

**Review by KKlimczukS — CHANGES_REQUESTED** [review]

File: devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy (hunk 1 above)
Comment (k-morawski) [reply devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy]: Dear @tklee2020 , could you please fix the indentations by change 4 spaces to 1 tab ? Many thanks

File: devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy [hunk 2]
```diff
@@ -33,6 +33,16 @@ metadata {
...
+	fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0006", outClusters: "0006, 000A, 0019", model: "E220-KR6N0Z1-HA", deviceJoinName: "eZEX Switch 1" //EZEX Switch 2
+	fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0006", outClusters: "0006, 000A, 0019", model: "E220-KR2N0Z2-HA", deviceJoinName: "eZEX Switch 1" //EZEX Switch 3
+	fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0006", outClusters: "0006, 000A, 0019", model: "E220-KR3N0Z2-HA", deviceJoinName: "eZEX Switch 1" //EZEX Switch 3
+	fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0006", outClusters: "0006, 000A, 0019", model: "E220-KR4N0Z2-HA", deviceJoinName: "eZEX Switch 1" //EZEX Switch 3
+	fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0006", outClusters: "0006, 000A, 0019", model: "E220-KR5N0Z2-HA", deviceJoinName: "eZEX Switch 1" //EZEX Switch 3
+	fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0006", outClusters: "0006, 000A, 0019", model: "E220-KR6N0Z2-HA", deviceJoinName: "eZEX Switch 1" //EZEX Switch 3
```
Comment (KKlimczukS) [inline devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy]: Please add one more tab here, so that those fingerprint would align with others.

File: devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy [hunk 3]
```diff
@@ -277,23 +289,31 @@ private getChildCount() {
...
 		case "E220-KR6N0Z0-HA":
+	case "E220-KR6N0Z1-HA":
+	case "E220-KR6N0Z2-HA":
 		case "ZB-SW06":
 		case "JZ-ZB-006":
 		case "SBM300Z6":
 			return 6
 		case "E220-KR2N0Z0-HA":
+	case "E220-KR2N0Z1-HA":
+	case "E220-KR2N0Z2-HA":
```
Comment (KKlimczukS) [inline devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy]: Please add one more tab to Your `case ` statements so that they would align with others.

**Review by KKlimczukS — CHANGES_REQUESTED** [review]

**Comment by KKlimczukS** [comment]
@tklee2020 We noticed cancelled certification requests for those devices. Was it intentional ? Are You going to re-submit them ?

File: devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy [hunk 4]
```diff
@@ -28,56 +28,109 @@ metadata {
...
-		fingerprint manufacturer: "LELLKI", model: "JZ-ZB-005", deviceJoinName: "LELLKI Switch 1" //LELLKI 5 Gang Switch 1
+		fingerprint manufacturer: "LELLKI", model: "JZ-ZB-005", deviceJoinName: "LELLKI Switch 1"
+		//LELLKI 5 Gang Switch 1
 		// Raw Description 01 0104 0100 00 05 0000 0003 0004 0005 0006 01 0000
-		fingerprint manufacturer: "LELLKI", model: "JZ-ZB-006", deviceJoinName: "LELLKI Switch 1" //LELLKI 6 Gang Switch 1
+		fingerprint manufacturer: "LELLKI", model: "JZ-ZB-006", deviceJoinName: "LELLKI Switch 1"
+		//LELLKI 6 Gang Switch 1
 		// SiHAS Switch (2~6 Gang)
 		fingerprint inClusters: "0000, 0003, 0006, 0019, ", outClusters: "0003,0004,0019", manufacturer: "ShinaSystem", model: "SBM300Z2", deviceJoinName: "SiHAS Switch 1"
 		fingerprint inClusters: "0000, 0003, 0006, 0019, ", outClusters: "0003,0004,0019", manufacturer: "ShinaSystem", model: "SBM300Z3", deviceJoinName: "SiHAS Switch 1"
```
Comment (KKlimczukS) [inline devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy:136]: Why did You add additional comments above the fingerprints not related to Your devices ?

File: devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy [hunk 5]
```diff
@@ -28,56 +28,109 @@ metadata {
...
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0006", outClusters: "0006, 000A, 0019", model: "E220-KR2N0Z2-HA", deviceJoinName: "eZEX Switch 1"
+		//EZEX Switch 3
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0006", outClusters: "0006, 000A, 0019", model: "E220-KR3N0Z2-HA", deviceJoinName: "eZEX Switch 1"
+		//EZEX Switch 3
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0006", outClusters: "0006, 000A, 0019", model: "E220-KR4N0Z2-HA", deviceJoinName: "eZEX Switch 1"
+		//EZEX Switch 3
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0006", outClusters: "0006, 000A, 0019", model: "E220-KR5N0Z2-HA", deviceJoinName: "eZEX Switch 1"
+		//EZEX Switch 3
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0006", outClusters: "0006, 000A, 0019", model: "E220-KR6N0Z2-HA", deviceJoinName: "eZEX Switch 1"
+		//EZEX Switch 3
```
Comment (KKlimczukS) [inline devicetypes/smartthings/zigbee-multi-switch.src/zigbee-multi-switch.groovy:60]: Why did You add such comments above Your fingerprints (for example: //EZEX Switch 3) while all fingerprints contain `deviceJoinName: "eZEX Switch 1"`

**Review by KKlimczukS — CHANGES_REQUESTED** [review]

**Comment by KKlimczukS** [comment]
> @tklee2020 We noticed cancelled certification requests for those devices. Was it intentional ? Are You going to re-submit them ?

@tklee2020 Since You added new commit recently, why those certification requests (in developer workspace) were cancelled before ?

**Comment by tklee2020** [comment]
> Since You added new commit recently, why those certification requests (in developer workspace) were cancelled before ?

I was not sure to change DTH by using only Graph API or both Graph API and Developer Workspace.
I will check the original source and make it back and add new devices and republish it.
